script/cypress-testrail-helper/index.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-console */
+const chalk = require('chalk');
+const clear = require('clear');
+const figlet = require('figlet');
+const fs = require('fs');
+
+const utils = require('./lib/utils');
+
+const appArgs = process.argv.slice(2);
+const myConfigPath = './script/cypress-testrail-helper/my-config.json'; // Project-root-relative path [for fs].
+
+// Runtime variables.
+let myConfig = undefined; // Config's generated & saved on first launch.
+let projectSwitching = false; // Flag for changing config's saved Project & Suite IDs.
+
+// Cypress run-specific config get/set function
+const configRunCySpec = appConfig => {
+  console.log(chalk.yellow('CREATING CYPRESS RUN-SPECIFIC CONFIG-FILE NOW...'));
+  utils
+    .getSetCypressConfig(appConfig)
+    .then(specPath => {
+      console.log(chalk.green('CYPRESS RUN-SPECIFIC CONFIG-FILE CREATED!'));
+      console.log(chalk.green(`NOW RUNNING CYPRESS SPEC: ${specPath}`));
+      console.log(
+        chalk.green(
+          'CLI-prompt will return after Cypress child-process exits.\nTHANKS FOR USING CYPRESS-TESTRAIL-HELPER!',
+        ),
+      );
+
+      // Run Cypress spec
+      utils.runCySpec(specPath);
+    })
+    .catch(err => {
+      console.error(chalk.red(`ERROR: ${err}`));
```

Comment:
We might want to add `process.exitCode = 1` here. `console.error` logs to `stderr`, but it does not set an exit code.